crates/uv-python/src/interpreter.rs

```diff
@@ -216,6 +217,21 @@ impl Interpreter {
         self.prefix.is_some()
     }
 
+    /// Returns `true` if this interpreter is managed by uv.
+    ///
+    /// Returns `false` if we cannot determine the path of the uv managed Python interpreters.
+    pub fn is_managed(&self) -> bool {
+        let Ok(installations) = ManagedPythonInstallations::from_settings() else {
+            return false;
+        };
+
+        installations
+            .find_all()
```

Comment:
Should we cache this? It's still faster than it needs to be, but it's still an io operation, so if we optimize this for speed, I'd cache this `read_dir` first.